docs/getting-started/javascript-options-api.md

```diff
@@ -104,22 +101,25 @@ Kendo UI for Vue includes [four artfully designed themes](slug:themesandstyles)
 1. In the `src/App.vue` file, add a `<script>` block and import the CSS files provided by the installed theme package:
  
     ```js
-      import '@progress/kendo-theme-default/dist/all.css';
+      <script>
+        import '@progress/kendo-theme-default/dist/all.css';
+      </script>
     ```
  
-You can add any additional custom styles in the `<styles>` tag of the `src/App.vue` file.
+To add any custom styles to your app, insert a `<styles>` tag in the `src/App.vue` file and place your styles there.
  
 ## Add a Vue Data Grid Component
  
 Now that you've installed all required packages, you are ready to add the Kendo UI for Vue Data Grid to the application:
  
-1. In the `src/App.vue` file, add a `<script>` block and import the Grid and its data:
+1. In the `<script>` block of the `src/App.vue` file, import the Grid and its data. In addition, the `process` function from the [Data Query](https://www.telerik.com/kendo-vue-ui/components/dataquery/) package will allow you to use data operations like sorting, paging, and filtering.
```

Comment:
I think it is better to say  will allow you to apply data operations like sorting, paging, and filtering`